static/app/views/issueList/filters.tsx

```diff
@@ -49,17 +49,6 @@ const StyledPageFilterBar = styled(PageFilterBar)`
   > div > button {
     width: 100%;
   }
-
-  & > * {
-    /* Prevent date filter from shrinking below 6.5rem */
-    &:nth-last-child(2) {
-      min-width: 6.5rem;
-    }
-
-    &:last-child {
-      min-width: 0;
```

Comment:
this selects the date filter, which already has a min width (in `pageFilterBar.tsx`)